python/tvm/relay/frontend/tensorflow.py

```diff
@@ -34,6 +34,17 @@
 
 __all__ = ['from_tensorflow']
 
+def _infer_value(input_val, params):
+    from tvm.contrib import graph_runtime
+    func = _expr.Function(ir_pass.free_vars(input_val), input_val)
+    with tvm.relay.build_config(opt_level=0):
```

Comment:
Why `opt_level=0`? Is it because fusion causes problem?

Reply:
This is actually just a helper function that I directly pulled out of reshape and always used opt_level 0. I think the reason is that the graph being built is purely to evaluate a value and will not be part of the output graph. Since performance doesn't matter, using opt_level 0 helps prevent issues like over fusing constants.

Reply:
I am fine with this as it is only needed by a few ops.